iocore/eventsystem/UnixEventProcessor.cc

```diff
@@ -395,6 +395,7 @@ EventProcessor::initThreadState(EThread *t)
 {
   // Run all thread type initialization continuations that match the event types for this thread.
   for (int i = 0; i < MAX_EVENT_TYPES; ++i) {
+    thread_group[i]._count_started++;
```

Comment:
prefix increment where possible. Good habits... Although it makes an actual performance difference on atomics.